docs/getting_started/configuration.rst

```diff
@@ -257,94 +244,192 @@ Here is a guideline to tune the effort corrections and friction parameters.
+
+Ranges:
+
+-   :member:`trossen_arm::Link::mass`: :math:`\mathbb{R}`
+-   :member:`trossen_arm::Link::inertia`: :math:`\mathbb{R}^9`
+-   :member:`trossen_arm::Link::origin_xyz`: :math:`\mathbb{R}^3`
+-   :member:`trossen_arm::Link::origin_rpy`: :math:`\mathbb{R}^3`
+
+Finger Offsets
+^^^^^^^^^^^^^^
+
+The offsets of the left and right fingers define the home position specific to the fingers.
+
+- :member:`trossen_arm::EndEffector::offset_finger_left`: the offset from the palm center to the left carriage center in m with the fingers closed
+- :member:`trossen_arm::EndEffector::offset_finger_right`: the offset from the palm center to the right carriage center in m with the fingers closed
+
+Ranges: :math:`\mathbb{R}`
+
+Pitch circle radius
+^^^^^^^^^^^^^^^^^^^
+
+:member:`trossen_arm::EndEffector::pitch_circle_radius` specifies pitch circle radius of the pinion of the end effector.
+
+Range: :math:`\mathbb{R}`
+
+t_flange_tool
+^^^^^^^^^^^^^
 
-    If you run into the :ref:`troubleshooting:16: robot input discontinuous` error state, please
+:member:`trossen_arm::EndEffector::t_flange_tool` defines the tool frame pose measured in the flange frame as shown in the image below.
 
-    - check that the frequency of the control loop is at least 300 Hz if the built-in interpolator is disabled, i.e., the ``goal_time`` is zero
-    - otherwise, check that the ``goal_time`` is long enough compared to looping time so that the joint can feasibly reach ``goal_position`` within ``goal_time``
-    - finally, if you're confident that there's no implementation mistake in the script, increase the continuity factor with discretion
+.. image:: images/t_flange_tool.jpeg
+    :align: center
+    :width: 600px
 
 .. note::
 
-    Continuity checks can be disabled by setting a joint's continuity factor to a negative value.
+    The first 3 elements are the translation and the last 3 elements are the angle-axis representation of the rotation
 
-    .. warning::
+Range: :math:`\mathbb{R}^6`
 
-        This is a potentially dangerous option as it allows users to command positions to joints that far exceed physical limits.
-        Please use it with caution.
+Joint Modes
+-----------
 
-End Effector
+The joint modes define the mode of operation of each joint.
+
+Choices: :enum:`trossen_arm::Mode`
+
+Joint Limits
 ------------
 
-The :class:`trossen_arm::EndEffectorProperties` allow the usage of different end effectors.
-It's important to match the end effector properties with the actual end effector attached to the arm.
-Otherwise, the controller won't be able to properly compensate for the end effector's weight and inertia.
+The joint limits define the operating limits of each joint.
 
-.. tip::
+The block diagram of the control loop of the motor is given below.
 
-    The commonly used :class:`trossen_arm::StandardEndEffector` are provided in the driver.
+.. mermaid::
```

Comment:
I love this diagram but I'm not sure that here in **config** > **joint limits** is the best place to put it.

Maybe we can have a concepts or theory of operation section?